ovos_config_assistant/app.py

```diff
@@ -43,13 +44,14 @@ def oauth_callback(oauth_id):
         data=body,
         auth=(client_id, client_secret),
     ).json()
-
-    with JsonStorageXDG("ovos_oauth") as db:
-        db.add_token(oauth_id, token_response)
+    print(oauth_id)
+    print(token_response)
+    data['token_response'] = token_response
+    oauth_store.store()
```

Comment:
the usage of the `with` statement handles this for you, i think it would be best to fix the db.add_token there to the proper method, that also removes the need for the oauth_store variable since it isnt referenced